CODEOWNERS

```diff
@@ -0,0 +1,25 @@
+# This file provides an overview of responsibilities in this repository.
+
+# Please note that this file does not represent all contributions to the code. What persons and organizations
+# actually contributed to each file can be seen on GitHub and is documented in the license headers.
+
+# Linked persons with write permissions are automatically added as reviewers when a pull request is opened.
+
+# Each line is a file pattern followed by one or more contact persons. The last matching pattern has the most precedence.
+# For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
+
+CONTRIBUTING.md @paullatzelsperger
+LICENSE @paullatzelsperger
+SECURITY.md @paullatzelsperger
+README.md @paullatzelsperger
+
+.github/actions/ @paullatzelsperger
+.github/ISSUE_TEMPLATE @paullatzelsperger
+.github/workflows/ @paullatzelsperger
+
+runtime-metamodel @paullatzelsperger @jimmarino
```

Comment:
As far as I remember, directory paths have to start and end with a "/" if you want to express that any containing files should be covered.

Reply:
mmh, ok, will add them

Reply:
see the file in the connector repo